libs/shared/contentful/src/lib/contentful.manager.ts

```diff
@@ -55,6 +60,22 @@ export class ContentfulManager {
     });
   }
 
+  async getEligibilityContentByOffering(offering: string) {
+    const queryResult = await this.client.query(
+      eligibilityContentByOfferingQuery,
+      {
+        skip: 0,
+        limit: 100,
+        locale: DEFAULT_LOCALE,
```

Comment:
These should no longer be necessary.

Out of curiosity, did typescript not have any warnings about this?

Reply:
No warnings from TS. I'll revise, thanks!